help/configure-who-can-invite-to-channels.md

```diff
@@ -4,7 +4,7 @@
 
 By default, anyone other than guests can add other users to channels. However,
 you can restrict the ability to do so to specific
-[roles](/help/roles-and-permissions). You can also can configure who can remove
+[roles](/help/user-roles). You can also can configure who can remove
 other users from a public channel. For private channels, administrators must be
 subscribed to the channel to configure these settings.
```

Comment:
Not for this PR, but this probably needs a rewrite following support for configuring a group that can administer channels.